hpcgap/src/gvars.c

```diff
@@ -583,17 +582,6 @@ UInt GVarName (
     return INT_INTOBJ(gvar);
 }
 
-Int FindGVarByValue(Obj value, Int start) {
-  Int max = CountGVars;
```

Comment:
My point is: this function was added but never used nor documented. It is also trivial to recreate. Based on past experience, this means it will never be used: anybody who needs it will never find this one, but instead create a similar function from scratch.